specutils/io/default_loaders/manga.py

```diff
@@ -53,7 +53,7 @@ def manga_cube_loader(file_obj, **kwargs):
 
     spaxel = u.Unit('spaxel', represents=u.pixel, doc='0.5" spatial pixel', parse_strict='silent')
     with read_fileobj_or_hdulist(file_obj, **kwargs) as hdulist:
-        spectrum = _load_manga_spectra(hdulist, per_unit=spaxel, transpose=True)
+        spectrum = _load_manga_spectra(hdulist, per_unit=spaxel)
```

Comment:
You mention the mask and uncertainty are now reshaped.  Is that why the `transpose` was removed?  The SDSS MaNGA files, including flux, is originally stored transposed to what `Spectrum1D` expected?  Does `Spectrum1D` fix this and transposes the entire cube internally?  I want to make sure we don't end up with transposed fluxes but correctly oriented masks and uncertainties.

Reply:
According to the the tests, the shape is what we would expect. It may be worth updating the [unit tests](https://github.com/astropy/specutils/blob/main/specutils/tests/test_loaders.py#L153) to also check the shape of the uncertainties and mask.

Reply:
Sorry, was going to reply to this earlier and got distracted. The transpose was needed because `Spectrum1D` requires the spectral axis to be the last axis of the flux array, and it was removed because now `Spectrum1D` always performs this check and reshapes the arrays (flux, uncertainty and mask) if needed in its initialization code. It also warns the user if it did this reshaping to give them a heads up that the `Spectrum1D` is shaped differently than the input data.

There is a subtle difference in how it's handled now, which is that rather than transposing the input arrays, `Spectrum1D` now uses `np.moveaxis` internally (because the WCS needs to be updated to match the arrays and the astropy WCS API only has an equivalent of `moveaxis`, with no transpose method). So previously if your input array axes were say `[a, b, c]` (with 'a' being the spectral axis) the resulting `Spectrum1D` would have axes `[c, b, a]`. Now the resulting `Spectrum1D` will have axes `[b, c, a]`.

Reply:
Thanks for the explanation.  This is helpful.  We may want to update the user-facing documentation with a section explaining what Spectrum1D now does to input data.  It might be confusing to users who load their data in and find it's been reoriented.  With maybe a toy example.